code/framework/src/scripting/engines/node/builtins/color_rgba.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * MafiaHub OSS license
+ * Copyright (c) 2021-2023, MafiaHub. All rights reserved.
+ *
+ * This file comes from MafiaHub, hosted at https://github.com/MafiaHub/Framework.
+ * See LICENSE file in the source repository for information regarding licensing.
+ */
+
+#pragma once
+
+#include "../../../keys.h"
+
+#include <glm/glm.hpp>
+#include <v8pp/class.hpp>
+#include <v8pp/module.hpp>
+
+#include <iomanip>
+#include <list>
+#include <sstream>
+
+namespace Framework::Scripting::Engines::Node::Builtins {
+    class ColorRGBA {
+      private:
+        glm::ivec4 _data;
+
+      public:
+        ColorRGBA(int r, int g, int b, int a) {
+            _data = {r, g, b, a};
+        }
+
+        int GetR() const {
+            return _data.r;
+        }
+
+        int GetG() const {
+            return _data.g;
+        }
+
+        int GetB() const {
+            return _data.b;
+        }
+
+        int GetA() const {
+            return _data.a;
+        }
+
+        float GetRelR() const {
```

Comment:
What does Rel stand for?

Reply:
`Rel` is for `Relative`.

Since RGB/RGBA use int (0-255) and game use float vector (0.0-1.0), this function convert int value to float.

I used `Rel` because that's how they name this kind of operation in the game, but it's bad practice to use acronyms.

Do you have suggestion, or should I rename it to `GetRelativeR`?

Reply:
I think use of `Float` keyword would make more sense here, but I will let you decide.